saturn-gpu/environment.yml

```diff
@@ -39,20 +39,19 @@ dependencies:
 - prefect
 - pyarrow
 - python=3.7
-- pytorch=1.4.0=py3.7_cuda10.1.243_cudnn7.6.3_0
-- rapids=0.15.1=cuda10.1_py37_gc1db54b_5
+- pytorch=1.7.0=py3.7_cuda10.1.243_cudnn7.6.3_0
+- rapids
 - s3fs
 - scikit-learn
 - scipy
 - snowflake-connector-python==2.3.2
 - snowflake-sqlalchemy
-- tensorboard=2.1.0
-- tensorflow-base=2.1.0=gpu_py37h6c5654b_0
-- tensorflow-estimator=2.1.0
-- tensorflow=2.1.0=gpu_py37h7a4bb67_0
-- torchvision=0.5.0=py37_cu101
+- tensorboard=2.2.1
+- tensorflow-estimator=2.2.0
+- tensorflow==2.2.0=gpu_py37h1a511ff_0
```

Comment:
I don't see the point of us putting tensorflow-base since it is pinned to the build in the package.